### PR TITLE
[15.0] [MIG] base_substate

### PR DESCRIPTION
**Pull request by alexeirivera87**
Hello

migration to v15

regards

### CODE REVIEW

**Comment by alexeirivera87** [comment]
Hello @mourad-ehm @ernestotejeda @sbidoul

**Review by dreispt — APPROVED** [review]

File: base_substate/models/base_substate_mixin.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def _get_substate_type(
         string="Sub State",
         ondelete="restrict",
         default=lambda self: self._get_default_substate_id(),
-        track_visibility="onchange",
+        tracking=5,
```
Comment (kittiu) [inline base_substate/models/base_substate_mixin.py:98]: Why tracking by order, and not just tracking=True? What is the benefit here?

**Review by kittiu — APPROVED** [review]
Functional Tested, LGTM

**Review by mourad-ehm — APPROVED** [review]

**Pull request merged**